drivers/docker/driver.go

```diff
@@ -1647,7 +1647,7 @@ func (d *Driver) DestroyTask(taskID string, force bool) error {
 
 	c, err := dockerClient.ContainerInspect(d.ctx, h.containerID)
 	if err != nil {
-		if _, ok := err.(errdefs.ErrNotFound); ok {
+		if errdefs.IsNotFound(err) {
```

Comment:
this was the test fix. the error type wasn't getting cast as expected anymore, so it thought it was some other kind of error and hit the else below.

moby's errdefs are deprecated as of https://github.com/moby/moby/commit/f1bb44aeee8fe115075fe06b588f5f4b8db471d8 and now merely point to containerd's